app/src/main/java/org/lineageos/twelve/datasources/MediaDataSource.kt

```diff
@@ -63,6 +63,30 @@ interface MediaDataSource {
 
     /**
      * Get the playlist information and all the tracks of the given playlist.
+     * If the playlist contains an audio that is unavailable, it will be mapped to null.
```

Comment:
Imo we should simply not map them in case.

Reply:
Users should have a chance to remove the entry even it it isn't available